orangecontrib/spectroscopy/io/__init__.py

```diff
@@ -23,3 +23,5 @@
 from .maxiv import HDRReader_STXM
 from .soleil import SelectColumnReader, HDF5Reader_HERMES, HDF5Reader_ROCK
 from .cls import HDF5Reader_SGM
+
+from .perkinelmer import PerkinElmerReader
```

Comment:
Let's put this in the `# Instrument-specific readers` section

Reply:
done